README.md

```diff
@@ -35,6 +35,12 @@ Run as a service, exposing ports 1521 (TCP database server) and 81 (web interfac
 docker run -d -p 1521:1521 -p 81:81 -v /path/to/local/data_dir:/opt/h2-data --name=MyH2Instance oscarfonts/h2
 ```
 
+Or run as a service with an extra custom config set in the command line, like allowing to create database at connection:
+
+```
+docker run -d -p 1521:1521 -p 81:81 -v /path/to/local/data_dir:/opt/h2-data -e H2_OPTIONS='-ifNotExists' --name=MyH2Instance oscarfonts/h2
```

Comment:
> the one who runs the container should take care of this

People should be warned how dangerous is that option. Otherwise most of them will not realize what they actually doing. Please, don't assume that people will take a look on H2's own documentation. They wouldn't.

It will be even better to exclude that option from example command line and describe it somewhere below together with warnings.

Reply:
Ok for me to remove this example from the readme.